setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.csproj

```diff
@@ -17,6 +17,10 @@
   <ItemGroup>
     <SwrFile Include="*.swr" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Move these references from all the projects to \build\import\VisualStudio.props

Reply:
done in next commit